pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -131,8 +131,6 @@ class ModelCheckpoint(Callback):
 
     CHECKPOINT_JOIN_CHAR = "-"
     CHECKPOINT_NAME_LAST = "last"
-    CHECKPOINT_STATE_BEST_SCORE = "checkpoint_callback_best_model_score"
-    CHECKPOINT_STATE_BEST_PATH = "checkpoint_callback_best_model_path"
```

Comment:
Just fyi, I added these some time ago because with raw strings, it happened once that there was a typo which introduced a bug. But everybody gradually removed them as it seems everybody wants to use raw strings for some reason. Maybe it adds clarity but at the same time it's danger imo.

Reply:
it used to be a problem when Trainer actually reached into the callbacks and did modifications from outside when checkpointing. but this is not the case anymore, so I guess it's ok to remove it as it is limited to this class.